src/careamics_restoration/bioimage/io.py

```diff
@@ -0,0 +1,102 @@
+from pathlib import Path
+from typing import Union
+
+import yaml
+from bioimageio.core import load_resource_description
+from bioimageio.core.build_spec import build_model
+from bioimageio.spec.model.raw_nodes import Model
+
+from careamics_restoration.config.config import (
+    Configuration,
+)
+
+PYTORCH_STATE_DICT = "pytorch_state_dict"
+CAREAMICS_CONFIG = "careamics_config.yml"
```

Comment:
Where is this used? And why?

Reply:
I should remove the `CAREAMICS_CONFIG`. It was for the previous way of exporting.